apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -466,9 +472,33 @@ export class LandingPage extends ViewPlugin {
           <div class="d-flex justify-content-between">
             <div class="d-flex flex-column">
               <div class="border-bottom d-flex justify-content-between clearfix py-3 mb-4">
-                <div class="mx-4 w-100">
+                <div class="mx-4 w-100 d-flex">
                   ${img}
                   <audio id="remiAudio" muted=false src="assets/audio/remiGuitar-single-power-chord-A-minor.wav"></audio>
+                  <div class="w-80 pl-5 ml-5">
+                    <h6 class="mb-1">Quicklinks</h6>
+                    <spam class="pl-0">
+                      <u class="${css.text} pr-1" onclick=${() => migrateWorkspace()}>Migrate File system</u>
+                      to workspace
+                    </spam>
+                    <p class="font-weight-bold mb-0">2-step solution:</p>
+                    <li class="pl-1">
+                      manually
+                      <u class="${css.text} px-1" onclick=${() => downloadFiles()}>Download all Files</u>
+                      as a backup zip
+                    </li>
+                    <li class="pl-1">
+                      use
```

Comment:
`u` of `use` can be capitalized

Reply:
Actually `use Restore Backup zip to current workspace` doesn't seem right. May we should remove `use`